functionary/ui/templates/base.html

```diff
@@ -45,124 +48,97 @@
+                </a>
+                <h2 class="navbar-brand">Run your functions</h2>
+            </div>
+        </nav>
+        <div class="container-fluid">
+            <div class="row flex-nowrap">
+                <div class="col-auto px-sm-2 px-0 shadow">
+                    <div class="d-flex flex-column justify-content-between px-3 pt-2 min-vh-100 max-vh-100">
+                        {% if user.is_authenticated %}
+                            <nav class="nav nav-pills flex-column mb-auto align-items-center align-items-md-start">
+                                {% include "partials/env_selector.html" %}
+                                <a class="nav-link px-0" href="{% url 'ui:task-list' %}">
+                                    <i class="fa fa-lg fa-clipboard-list text-secondary"></i>
+                                    <span class="ms-1 d-none d-md-inline">Tasking</span>
+                                </a>
+                                <a class="nav-link px-0" href="{% url 'ui:build-list' %}">
+                                    <i class="fa fa-lg fa-wrench text-secondary"></i>
+                                    <span class="ms-1 d-none d-md-inline">Builds</span>
+                                </a>
+                                <a class="nav-link px-0" href="{% url 'ui:package-list' %}">
+                                    <i class="fa fa-lg fa-cubes text-secondary"></i>
+                                    <span class="ms-1 d-none d-md-inline">Packages</span>
+                                </a>
+                                <a class="nav-link px-0" href="{% url 'ui:function-list' %}">
+                                    <i class="fa fa-lg fa-cube text-secondary"></i>
+                                    <span class="ms-1 d-none d-md-inline">Functions</span>
+                                </a>
+                                <a class="nav-link px-0" href="{% url 'ui:workflow-list' %}">
+                                    <i class="fa fa-lg fa-diagram-next text-secondary"></i>
+                                    <span class="ms-1 d-none d-md-inline">Workflows</span>
+                                </a>
+                                <a class="nav-link px-0" href="{% url 'ui:scheduledtask-list' %}">
+                                    <i class="fa fa-lg fa-clock text-secondary"></i>
+                                    <span class="ms-1 d-none d-md-inline">Schedules</span>
+                                </a>
+                                {% if user.is_superuser %}
+                                    <a class="nav-link px-0" href="{% url 'ui:environment-list' %}">
+                                        <i class="fa fa-lg fa-building text-secondary"></i>
+                                        <span class="ms-1 d-none d-md-inline">Environments</span>
+                                    </a>
+                                    <a class="nav-link px-0" href="{% url 'ui:team-list' %}">
```

Comment:
Small thing that I note here since this is where it's most noticeable...

The alignment of the menu text is off.  With the previous styling, the icons were all centered with each other and then there was consistent spacing so that the text was also all aligned.  Now the icons are all left-aligned with the text not really aligned at all.  The bootstrap [sidebars example](https://getbootstrap.com/docs/5.0/examples/sidebars/#) shows alignment similar to what we had, so it seems like it should be doable.